scripts/deploy_oak_functions_loader

```diff
@@ -7,8 +7,7 @@ source "$SCRIPTS_DIR/common"
 # shellcheck source=scripts/gcp_common
 source "$SCRIPTS_DIR/gcp_common"
 
-# TODO(#1943): Remove when #1943 is submitted.
```

Comment:
The point of the TODO is to actually use a runner command when implemented, instead of cargo build (not just to remove the TODO) :slightly_smiling_face: (also below)

Reply:
Ah! Fixed.